src/pe/options.rs

```diff
@@ -8,6 +8,16 @@ pub struct ParseOptions {
     /// memory](https://learn.microsoft.com/en-us/windows/win32/debug/pe-format#other-contents-of-the-file).
     /// For on-disk representations, leave as true. Default: true
     pub parse_attribute_certificates: bool,
+    /// Whether or not to end with an error in case of incorrect data or continue parsing if able. Default: ParseMode::Strict
```

Comment:
debating whether we should add `#[non_exhaustive]` to this struct to make it future compatible if we need to add another field like this in the future (and not break people)